ERCS/erc-4337-rpc.md

```diff
@@ -0,0 +1,542 @@
+---
+eip:
+title: JSON-RPC API for Account Abstraction Using the UserOperation Mempool
+description: A set of JSON-RPC API methods that defines a communication between smart contract account wallets and bundlers
+author: Vitalik Buterin (@vbuterin), Yoav Weiss (@yoavw), Dror Tirosh (@drortirosh), Shahaf Nacson (@shahafn), Alex Forshtat (@forshtat)
+discussions-to:
```

Comment:
Please create a discussions topic in Eth Magicians.  It just needs to be a link to this PR.